docs/openapi/schemas.yaml

```diff
@@ -426,6 +427,12 @@ AuditResult:
           description: A list of experimentation opportunities
           items:
             $ref: './schemas.yaml#/ExperimentationOpportunity'
+    - type: object
+      properties:
+        urlInspection:
```

Comment:
This wrapper could be skipped and you could directly include: `index(ability)`, `richResults` with their properties.